examples/js/controls/FirstPersonControls.js

```diff
@@ -295,6 +313,20 @@ THREE.FirstPersonControls = function ( object, domElement ) {
 
 	}
 
+	function setOrientation( controls ) {
+
+		var quaternion = controls.object.quaternion;
+
+		lookDirection.copy( controls.forward ).applyQuaternion( quaternion );
```

Comment:
As I said previously, 
```javascript
lookDirection.set( 0, 0, - 1 ).applyQuaternion( quaternion );
```
This control only supports cameras.

Reply:
I have tested it and there are some limitations, yes. 

Demo: https://jsfiddle.net/q50o1r97/1/

It would be possible with a few modification, though. But for now, I hardwire the forward vector.